build-utils/css/constants/typography.ts

```diff
@@ -0,0 +1,43 @@
+import {} from './utils';
```

Comment:
Is there a reason for this empty import?

Reply:
That's my mistake when cleaning up. Good catch! Thanks